asv_bench/benchmarks/plotting.py

```diff
@@ -99,4 +106,33 @@ def time_plot_andrews_curves(self):
         andrews_curves(self.df, "Name")
 
 
+class BackendLoading:
+    repeat = 1
+    number = 1
+    warmup_time = 0
+
+    def setup(self):
+        dist = pkg_resources.get_distribution("pandas")
+        if dist.module_path not in pd_file:
```

Comment:
what is the purpose here? this is very confusing

Reply:
I copied what's in `test_backend`, introduced in #27810, to skip the benchmark (when it raises) if the branch is not installed. But maybe it's pointless here?

Reply:
yeah i think you can remove this